### ISSUE TITLE
Image refresh for continuous-atomic

### ISSUE THREAD

**Issue by cockpituous** [cockpit-project/bots#871]
Image refresh for continuous-atomic
 * [ ] FAIL: image-refresh continuous-atomic

**Comment by cockpituous** [comment]
image-refresh in progress on 1-ci-srv-01.
Log: https://logs.cockpit-project.org/logs/image-refresh-871-20200516-234040/

**Comment by cockpituous** [comment]
Task failed: https://logs.cockpit-project.org/logs/image-refresh-871-20200516-234040/

**Comment by marusak** [comment]
So there is a new trend, currently at least 4 imagre refreshes failed with:
```
+ /var/lib/testvm/zero-disk.setup --keep-mock-cache
journalctl: unrecognized option '--relinquish-var'
libvirt: QEMU Driver error : Requested operation is not valid: domain is not running
Running setup script /build/cockpit-project/bots/make-checkout-workdir/images/scripts/rhel-8-2.setup
+ ! test -f /run/nologin && cat /proc/sys/kernel/random/boot_id
Uploading /build/cockpit-project/bots/make-checkout-workdir/images/scripts/lib
scp -B -r -p -P 2201 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o ControlPath=/build/tmp/ssh-%h-%p-%r-26896 -o BatchMode=yes /build/cockpit-project/bots/make-checkout-workdir/images/scripts/lib root@[127.0.0.2]:/var/lib/testvm
Uploading /build/cockpit-project/bots/make-checkout-workdir/images/scripts/rhel-8-2.setup
scp -B -r -p -P 2201 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o ControlPath=/build/tmp/ssh-%h-%p-%r-26896 -o BatchMode=yes /build/cockpit-project/bots/make-checkout-workdir/images/scripts/rhel-8-2.setup root@[127.0.0.2]:/var/tmp/SETUP
Uploading /build/cockpit-project/bots/make-checkout-workdir/images/scripts/lib/base
scp -B -r -p -P 2201 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o ControlPath=/build/tmp/ssh-%h-%p-%r-26896 -o BatchMode=yes /build/cockpit-project/bots/make-checkout-workdir/images/scripts/lib/base root@[127.0.0.2]:/var/tmp/cockpit-base
run setup script on guest
killing ssh master process 27024
Traceback (most recent call last):
  File "/build/cockpit-project/bots/make-checkout-workdir/image-create", line 102, in run_setup_script
    environment=env, quiet=not self.machine.verbose, timeout=7200)
  File "/build/cockpit-project/bots/make-checkout-workdir/machine/machine_core/ssh_connection.py", line 316, in execute
    ret = select.select(rset, wset, [], 10)
  File "/build/cockpit-project/bots/make-checkout-workdir/machine/machine_core/timeout.py", line 41, in handle_timeout
    raise RuntimeError(self.error_message)
RuntimeError: Timed out on '<script>'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/build/cockpit-project/bots/make-checkout-workdir/image-create", line 197, in <module>
    builder.build()
  File "/build/cockpit-project/bots/make-checkout-workdir/image-create", line 139, in build
    self.run_setup_script(script)
  File "/build/cockpit-project/bots/make-checkout-workdir/image-create", line 117, in run_setup_script
    self.machine.stop(timeout_sec=500)
  File "/build/cockpit-project/bots/make-checkout-workdir/machine/machine_core/machine_virtual.py", line 512, in stop
    self.shutdown(timeout_sec=timeout_sec)
  File "/build/cockpit-project/bots/make-checkout-workdir/machine/machine_core/machine_virtual.py", line 575, in shutdown
    self._domain.shutdown()
  File "/usr/lib64/python3.7/site-packages/libvirt.py", line 2506, in shutdown
    if ret == -1: raise libvirtError ('virDomainShutdown() failed', dom=self)
libvirt.libvirtError: Requested operation is not valid: domain is not running
```

**Comment by cockpituous** [comment]
image-refresh in progress on 1-ci-srv-04.
Log: https://logs.cockpit-project.org/logs/image-refresh-871-20200519-081111/

**Comment by cockpituous** [comment]
Task failed: https://logs.cockpit-project.org/logs/image-refresh-871-20200519-081111/

**Comment by cockpituous** [comment]
image-refresh in progress on 2-cockpit-7.
Log: https://logs.cockpit-project.org/logs/image-refresh-871-20200521-073535/

**Comment by cockpituous** [comment]
Task failed: https://logs.cockpit-project.org/logs/image-refresh-871-20200521-073535/

**Comment by cockpituous** [comment]
image-refresh in progress on centosci-tasks-7w9qz.
Log: https://logs-https-cockpit.apps.ci.centos.org/logs/image-refresh-871-20200525-145454/

**Comment by cockpituous** [comment]
Task failed: https://logs-https-cockpit.apps.ci.centos.org/logs/image-refresh-871-20200525-145454/

**Comment by cockpituous** [comment]
image-refresh in progress on 2-ci-srv-05.
Log: https://logs.cockpit-project.org/logs/image-refresh-871-20200526-044848/

**Comment by cockpituous** [comment]
Task failed: https://logs.cockpit-project.org/logs/image-refresh-871-20200526-044848/

**Comment by martinpitt** [comment]
It looks like they time out during dd filling the disks with zero. That could be due to overflowing /tmp/, I'm currently running a build manually in a bot and check what happens.

**Comment by martinpitt** [comment]
Fix sent in PR #912
